src/pdf/pdf-view.js

```diff
@@ -519,6 +522,8 @@ class PDFView {
 		for (let page of this._pages) {
 			if (!pageIndexes || pageIndexes.includes(page.pageIndex)) {
 				page.render();
+				// Aria label with page index set by pdf.js is off by one, so fix it here
+				page.originalPage.div.dataset.l10nArgs = JSON.stringify({ page: this._getPageLabel(page.pageIndex) });
```

Comment:
I noticed that each page has the aria-label of essentially the next page. So each page label needs to be decremented to be correct. The PDF reader accessibility setup is actually pretty good, as each page is a "region" and you can move virtual cursor from one page to another with standard screen reader shortcuts. When it happens, the aria-label of the page gets announced, and it's an issue because in the example below, you'll hear "Page 42", press arrowDown and it'll begin reading what's actually page 41, and so on.

Is it better to handle this elsewhere? Should I revert this?

<img width="1508" alt="Screenshot 2024-09-06 at 9 01 35 AM" src="https://github.com/user-attachments/assets/b363037c-4e86-4114-a1bf-78f661fec97a">

Reply:
We definitely want the page label because it is what you hear when the screen reader reads the page. So I think page label provides the most context and is a better way to identify the page.

I am not sure about page index... On one hand, it might add some additional info as to where in the document you are, so why not include it as well? On the other hand, maybe having two different identifications of a page will be distracting and a bit confusing?

Why don't we include just the page label for now to keep it simple, and I will make sure to add aria properties to page edit input in the toolbar to have the page index and total page count announced there. For casual navigation within the document, page label should be good but if you are wondering about the page index (which is something you need less often), you will check on the input in the toolbar. How does that sound?